aten/src/ATen/native/cuda/linalg/BatchLinearAlgebraLib.h

```diff
@@ -8,12 +8,8 @@
 #include <ATen/native/TransposeType.h>
 #include <ATen/native/cuda/MiscUtils.h>
 
-#if defined(CUDART_VERSION) && defined(CUSOLVER_VERSION)
-#define USE_CUSOLVER
-#endif
-
-#if defined(USE_ROCM) && ROCM_VERSION >= 50300
-#define USE_HIPSOLVER
+#if (defined(CUDART_VERSION) && defined(CUSOLVER_VERSION)) || (defined(USE_ROCM) && ROCM_VERSION >= 50300)
+#define USE_LINALG_SOLVER
```

Comment:
@malfet Does this address the macro-naming concerns you had?

Reply:
Yes, it does, thank you!